JASP-Engine/JASP/R/LSbinomialcommon.R

```diff
@@ -594,20 +594,21 @@
 }
 .marginalCustomBinomialLS   <- function(density, spikes, lCI, uCI, densityDiscrete = FALSE){
   
-  if (!is.null(density))
-    if (!densityDiscrete)density$y <- density$y/nrow(density)    
-    else
-      density <- data.frame("y" = NULL, "x" = NULL)
-    
-    if (length(spikes) != 0){
-      for(i in 1:length(spikes)){
-        density <- rbind(density[density$x <= spikes[[i]]$x,], spikes[[i]], density[spikes[[i]]$x < density$x,])
-      }
+  if (!is.null(density)){
+    if (!densityDiscrete)
+      density$y <- density$y/nrow(density)    
+  }else
+    density <- data.frame("y" = NULL, "x" = NULL)
+  
+  if (length(spikes) != 0){
+    for(i in 1:length(spikes)){
+      density <- rbind(density[density$x <= spikes[[i]]$x,], spikes[[i]], density[spikes[[i]]$x < density$x,])
     }
```

Comment:
```suggestion
  for(i in seq_along(spikes)) {
    density <- rbind(density[density$x <= spikes[[i]]$x,], spikes[[i]], density[spikes[[i]]$x < density$x,])
  }
```

Reply:
if `length(spikes) == 0`, `seq_along(spikes)` gives you `numeric(0)` and the loop is skipped, no need to nest it like this